docs/Quick-Start-Guide/Architecture.md

```diff
@@ -107,6 +107,45 @@ mirror:
 # gluster volume create test-volume replica 3 transport tcp server1:/exp1 server2:/exp2 server3:/exp3 server4:/exp4 server5:/exp5 server6:/exp6
 volume create: test-volume: success: please start the volume to access data
 ```
+4. **Dispersed Glusterfs Volume** - Dispersed volumes are based on
+erasure codes. It stripes the encoded data of files, with some redundancy added,
+across multiple bricks in the volume. You can use dispersed volumes to
+have a configurable level of reliability with minimum space waste.
+The number of redundant bricks in the volume can be decided by clients while
+creating the volume. Redundant bricks determines how many bricks can be lost
+without interrupting the operation of the volume.
+
+Create a dispersed volume:
+
+```
+# gluster volume create test-volume [disperse [<COUNT>]] [disperse-data <COUNT>] [redundancy <COUNT>] [transport tcp | rdma | tcp,rdma] <NEW-BRICK>
+```
+
+**For example**, three node dispersed volume with level of redundancy 1, (2 + 1):
+
+```console
+# gluster volume create test-volume disperse 3 server1:/exp1 server2:/exp2 server3:/exp3
+volume create: test-volume: success: please start the volume to access data
+```
+
+5. **Distributed Dispersed Glusterfs Volume** - Distributed dispersed volumes
+are the equivalent to distributed replicated volumes, but using dispersed subvolumes
+instead of replicated ones. The number of bricks must be a multiple of the 1st subvol.
+The purpose for such a volume is to easily scale the volume size and distribute the load
+across various bricks.
+
+Create a distributed dispersed volume:
+
+```
+# gluster volume create [disperse [<COUNT>]] [disperse-data <COUNT>] [redundancy <COUNT>] [transport tcp | rdma | tcp,rdma] <NEW-BRICK>
+```
+
+**For example**, six node distributed dispersed volume with level of redundancy 1, 2 x (2 + 1) = 6:
+
+```console
+# gluster volume create test-volume disperse 3 server1:/exp1 server2:/exp2 server3:/exp3 server4:/exp4 server5:/exp5 server6:/exp6
```

Comment:
Don't we need to add either disperse-data or redundancy here?

Reply:
No. All the three options(disperse, disperse-data and redundancy) can be used independently as well as in combination with each other. So this will work perfectly fine.

Reply:
It can be used independently, however, it is worth mentioning the default behaviour. I think if you just mention disperse count, it will calculate best redundancy count on its own (which is 50% I think).
So, it would be good to mention this and also recommend that user provide disperse-data and redundancy count as per their need.

Reply:
Done